app/src/main/java/com/keylesspalace/tusky/MainActivity.kt

```diff
@@ -76,6 +75,7 @@ import com.keylesspalace.tusky.components.login.LoginActivity
 import com.keylesspalace.tusky.components.preference.PreferencesActivity
 import com.keylesspalace.tusky.components.scheduled.ScheduledStatusActivity
 import com.keylesspalace.tusky.components.search.SearchActivity
+import com.keylesspalace.tusky.components.systemnotifications.NotificationService
```

Comment:
I'm open for a better name.

It is no "service" in the java sense. Only in the DI sense.

Reply:
I think it is fine